torch/csrc/cuda/shared/nvtx.cpp

```diff
@@ -42,8 +45,10 @@ static void* device_nvtxRangeStart(const char* msg, std::intptr_t stream) {
   RangeHandle* handle = (RangeHandle*)calloc(sizeof(RangeHandle), 1);
   handle->msg = strdup(msg);
   handle->id = 0;
-  cudaLaunchHostFunc(
-      (cudaStream_t)stream, device_callback_range_start, (void*)handle);
+  TORCH_CHECK(
```

Comment:
Is `C10_CUDA_CHECK` not preferred here for some reason?

Reply:
Switching to it.